mobile_app_connector/models/request.py

```diff
@@ -54,7 +53,7 @@ def mobile_contact_us(self, parameters, **params):
             'email_from': email,
             'subject': subject,
             'code':
-                self.env.ref('crm_claim_code.sequence_claim_app')
+                self.env.ref('sequence_claim_app')
                     .sudo()._next(),
```

Comment:
'mobile_app_connector.sequence_claim_app'
Otherwise it can't find the record and raises an exception